src/main/java/io/nats/client/BaseConsumeOptions.java

```diff
@@ -40,13 +41,15 @@ public class BaseConsumeOptions implements JsonSerializable {
     protected final int messages;
     protected final long bytes;
     protected final long expiresIn;
-    protected final long idleHeartbeat;
     protected final int thresholdPercent;
-    protected final boolean noWait;
+    protected final long idleHeartbeat;
+    protected final String group;
+    protected final long minPending;
+    protected final long minAckPending;
     protected final boolean raiseStatusWarnings;
 
-    @SuppressWarnings("rawtypes") // Don't need the type of the builder to get its vars
-    protected BaseConsumeOptions(Builder b) {
+    protected BaseConsumeOptions(Builder<?, ?> b) {
+        // Message / bytes is part of base and is calculated
```

Comment:
This isn't 2.11 specific, but I had just made changes in this area, then realized I had better changes uncommitted, so put them here.